### PR TITLE
fix: related products: actions for add to cart/wishlist

### PR DESCRIPTION
**Pull request by zoltantakacs-born**
The action buttons on the related product cards should work as expected.

## Related Issue
VBC-350

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: vetur.config.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+// vetur.config.js
+/** @type {import('vls').VeturConfig} */
+module.exports = {
```
Comment (scernisevs) [inline vetur.config.js:3]: hi @zoltantakacs-born ,
Could i please ask you to remove the file from the PR?
vetur it's a great tool but i don't think that everyone is using the extension on this project

**Review by dhargitai — APPROVED** [review]
Looks good

File: packages/theme/components/RelatedProducts.vue [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+    const {
+      wishlist,
+      load: loadWishlist,
+      addItem: addItemToWishlist,
+      isInWishlist,
+      removeItem: removeItemFromWishlist
+    } = isAuthenticated.value ? useWishlist() : useGuestWishlist();
+    const wishlistHelpers = useWishlistData();
+
+    onMounted(() => {
```
Comment (WojtekTheWebDev) [inline packages/theme/components/RelatedProducts.vue:92]: After [this PR](https://github.com/vuestorefront/bigcommerce/pull/123) you don't need to load the wishlist in the component, default layout is handling that

**Review by WojtekTheWebDev — CHANGES_REQUESTED** [review]
Good job, small remark from me in the comment below

**Review by scernisevs — APPROVED** [review]

**Review by WojtekTheWebDev — APPROVED** [review]
LGMT

**Pull request merged**